xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-test/xwiki-platform-extension-test-pageobjects/src/main/java/org/xwiki/extension/test/po/SearchResultsPane.java

```diff
@@ -71,10 +71,10 @@ public String getNoResultsMessage()
      */
     public ExtensionPane getExtension(String name, String version)
     {
-        String nameAndVersion = name + " " + version;
-        By xpath =
-            By.xpath("//form[contains(@class, 'extension-item') and descendant::*[contains(@class, "
-                + "'extension-title') and normalize-space(.) = '" + nameAndVersion + "']]");
+        String selector = String.format("//form[contains(@class, 'extension-item') and descendant::*["
+            + "contains(@class, 'extension-name') and normalize-space(.) = '%s'] and descendant::*["
+            + "contains(@class, 'extension-version') and normalize-space(.) = '%s']]", name, version);
```

Comment:
Same as above, you could match a dependency, so you need to restrict the search scope.